lighthouse-cli/test/smokehouse/test-definitions/a11y/expectations.js

```diff
@@ -691,16 +691,23 @@ const expectations = [
           },
         },
         'frame-focusable-content': {
-          score: 0,
-          details: {
-            items: [
-            ],
-          },
+          score: null,
+          scoreDisplayMode: 'notApplicable',
```

Comment:
I cannot get this audit to fail, no matter what I do. There are failing cases in the PR that introduced the rule (https://github.com/dequelabs/axe-core/pull/2785), but even verbatim copy/pasting them doesn't seem to trigger it. When I step through axe-core, it appears the `frameFocusableContentMatches` never even gets to see the iframes, only the top-level `html` element is passed in, so I'm not sure what's going on.

(notably the `dbw_tester` version is causing a passing audit in sample_v2, but not clear why that one isn't `notApplicable` as well)

We should probably figure out an actual failure case and/or file a bug on why the axe test can fail but we can't (it's late, maybe I'm just missing something), but in the meantime, should we not include in the LH 8 release?